app/src/main/java/com/rage/clamber/Activities/LoginActivity.java

```diff
@@ -64,71 +63,107 @@ public void onExistingUserButtonClicked (Button button) {
 
     }
 
+    /**
+     * Method called from ExistingUserLoginFragment when a username is entered. The method makes a
+     * call to the server to determine if the username exists in the database. If it does, the user
+     * is logged into the application and brought to the home page. The User is stored in the Home
+     * Page class in order to pull up other data related to that User in various fragments.
+     * @param user - username that has been entered in dialog fragment.
+     */
+
     public void existingUserPositiveClick(String user) {
         userName = user;
         if (userName.length() == 0) {
-            Toast.makeText(this, "Please enter a user name", Toast.LENGTH_SHORT).show();
+            Toast.makeText(this, R.string.enter_username, Toast.LENGTH_SHORT).show();
         } else {
             //Check for internet connectivity
             ConnectivityManager connectivityManager = (ConnectivityManager) this.getSystemService(Context.CONNECTIVITY_SERVICE);
             NetworkInfo networkInfo = connectivityManager.getActiveNetworkInfo();
             if (networkInfo != null && networkInfo.isConnected()) {
-                loginAsyncTask = new LoginAsyncTask(this);
-                loginAsyncTask.execute(userName);
+                Retrofit retrofit = new Retrofit.Builder()
```

Comment:
For the next PR, let's look into pulling the Retrofit Builder code out to a common spot, probably a Singleton object, as mentioned here: http://stackoverflow.com/questions/20579185/is-there-a-way-to-reuse-builder-code-for-retrofit
That should avoid having to re-do this code everywhere you need retrofit.